doc/users/faq/howto_faq.rst

```diff
@@ -9,6 +9,61 @@ How-to
 .. contents::
    :backlinks: none
 
+
+.. _how-to-too-many-ticks:
+
+Why do I have so many ticks, and/or why are they out of order?
+--------------------------------------------------------------
+
+One common cause for unexpected tick behavior is passing a list of strings
+instead of numbers or datetime objects. This can easily happen without notice
+when reading in a comma-delimited text file. Matplotlib treats lists of strings
+as "categorical" variables
+(:doc:`/gallery/lines_bars_and_markers/categorical_variables`), and by default
+puts one tick per "category", and plots them in the order in which they are
+supplied.
+
+.. plot::
+    :include-source:
+    :align: center
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    fig, ax = plt.subplots(1, 2, constrained_layout=True, figsize=(6, 2))
+
+    ax[0].set_title('Ticks seem out of order / misplaced')
+    x = ['5', '20', '1', '9']  # strings
+    y = [5, 20, 1, 9]
+    ax[0].plot(x, y, 'd')
+    ax[0].tick_params(axis='x', labelcolor='red', labelsize=14)
+
+    ax[1].set_title('Many ticks')
+    x = [str(xx) for xx in np.arange(100)]  # strings
+    y = np.arange(100)
+    ax[1].plot(x, y)
+    ax[1].tick_params(axis='x', labelcolor='red', labelsize=14)
+
+The solution is to convert the list of strings to numbers or
+datetime objects (often ``np.asarray(['2', '5', '1'], dtype='float')`` or::
+
+    np.asarray(['2021-10-01', '2021-11-02', '2021-12-03'],
+               dtype='datetime64[s]')
```

Comment:
Let's keep the formatting of both symmetric (not one inline and one as code block). Also it may make it more clear when using a descriptive variable name instead of example data.

```suggestion
datetime objects (often ``np.asarray(numeric_strings, dtype='float')`` or
``np.asarray(datetime_strings, dtype='datetime64[s]')``).
```